NEWS.md

```diff
@@ -1,4 +1,10 @@
 ## formatters 0.4.1.9001
+ * font_size is now 12 across the board, whereas before it was inconsistent between 8 and 12
```

Comment:
```suggestion
 * `font_size` is now 12 across the board, instead of being inconsistent between 8 and 12.
```
from GitHub Actions
/ grammar

NEWS.md#L2

"it was" is wordy or unneeded

Reply:
hopefully it works